TESTING_GUIDE.md

```
@@ -0,0 +1,324 @@
+Check that the event utilities are properly exported from the SDK:
+
+```bash
+cd c:\Users\HomePC\Documents\D\stellar_client_os
+
+# View the updated index.ts
+type packages/sdk/src/index.ts | findstr "events"
+```
+
+**Expected Output:** Should contain:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)